python/dask_cudf/dask_cudf/backends.py

```diff
@@ -113,6 +119,86 @@ def make_meta_cudf_index(x, index=None):
     return x[:0]
 
 
+def _empty_series(name, dtype, index=None):
+    if isinstance(dtype, str) and dtype == "category":
+        return cudf.Series([UNKNOWN_CATEGORIES], name=name, index=index).iloc[
+            :0
+        ]
```

Comment:
This Series wouldn't be of "Category" dtype, is that intended?

Reply:
Fixed it, sorry this was missed.